packages/eslint-config-expo/.eslintrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  root: true,
+  extends: ['./default.js', 'prettier'],
+  plugins: ['prettier'],
+  env: {
+    node: true,
+  },
+  rules: {
+    'prettier/prettier': 'error',
```

Comment:
I know this is just for internal use but prefer "warn" here so that formatting errors draw less attention and are less alarming (yellow squigglies) than errors that will prevent the code from running correctly (red squigglies)
```suggestion
    'prettier/prettier': 'warn',
```